game/src/main/scala/io/aigar/game/Cell.scala

```diff
@@ -97,6 +96,10 @@ class Cell(val id: Int, val player: Player, var position: Vector2 = new Vector2(
     Cell.radius(mass)
   }
 
+  override def scoreModification(): Float = {
+    (1 + log(mass) - log(Cell.MinMass)).toFloat
```

Comment:
Where is it used? I'm not sure but looks like this is not called by `Player` to reward itself after having eaten a `Cell`...